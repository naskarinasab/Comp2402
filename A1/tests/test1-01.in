Alfonso
Bernard   
Dino
Jude
Otha  
Dusty   
Lino   
Salvatore   

Walker   
Elden   
Scott   
Lee   
Tory   
Morris   
Ron  
Jack   
Clyde  
Brendon   
Morton   
Ruben   
Diego   
Norbert   
Edmundo   
Jamie   
Herschel   
Bradley
Vincenzo
Jaime
Alfredo
Rickie 
Teodoro
Alphonse
Chet   
Eugene
Brandon   
Abraham
Ernie 
Demarcus  
Rudolf  
Calvin 
Dean 
Toby 
Kerry
Casey 
Douglas 
Ronny  
Jarvis 
Ashley
Gilbert
