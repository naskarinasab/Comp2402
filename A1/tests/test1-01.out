Gilbert
Ashley
Jarvis 
Ronny  
Douglas 
Casey 
Kerry
Toby 
Dean 
Calvin 
Rudolf  
Demarcus  
Ernie 
Abraham
Brandon   
Eugene
Chet   
Alphonse
Teodoro
Rickie 
Alfredo
Jaime
Vincenzo
Bradley
Herschel   
Jamie   
Edmundo   
Norbert   
Diego   
Ruben   
Morton   
Brendon   
Clyde  
Jack   
Ron  
Morris   
Tory   
Lee   
Scott   
Elden   
Walker   

Salvatore   
Lino   
Dusty   
Otha  
Jude
Dino
Bernard   
Alfonso
